Singleton.py

```diff
@@ -0,0 +1,6 @@
+class Singleton(type):
+    _instances = {}
+    def __call__(cls, *args, **kwargs):
```

Comment:
[please add a blank line between class definition and method](https://www.python.org/dev/peps/pep-0008/#blank-lines)